ui/packages/app/src/project_setting/form/Stream.js

```diff
@@ -37,8 +37,7 @@ export const Stream = ({
         value={stream}
         options={streamOptions}
         onChange={onStreamChange}
-        onCreateOption={onStreamChange}
-        isDisabled={isDisabled}
```

Comment:
Can we not remove this `isDisabled` prop? 😅 Thanks 🙏

Reply:
fixed